libs/net/dal/Services/FolderService.cs

```diff
@@ -51,6 +51,7 @@ public IEnumerable<Folder> FindMyFolders(int userId)
         return this.Context.Folders
             .Include(f => f.Owner)
             .Include(f => f.ContentManyToMany)
+            .Include(f => f.Events).ThenInclude(f => f.Schedule)
```

Comment:
The problem with this approach is now every time we ask for content in a folder it will return events and schedules.  Which are not required 90% of the time.

Reply:
I suppose since it's just for FindMyFolders it'll be okay.